Exercise_D and E/assets/style/index.css

```diff
@@ -1,4 +1,69 @@
-/* 
-This css sheet is linked to practice.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+.top{
+    height: 60px;
+    padding-left: 20px;
+}
+.middle{
+    height: 700px;
+    background-image: url("/code/FTWD_Web_Day_02/Exercise_D and E/assets/images/flowershop.jpg");
```

Comment:
This looks like its the wrong path? it could be shorter? 
"../images/flowershop.jpg"